ApiReview/API.Polly.Core/NoDocs/Polly/Outcome.TResult.cs

```diff
@@ -0,0 +1,18 @@
+// Assembly 'Polly.Core'
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.ExceptionServices;
+
+namespace Polly;
+
+public readonly struct Outcome<TResult>
+{
+    public Exception? Exception { get; }
+    public TResult? Result { get; }
+    public bool HasResult { get; }
+    public bool IsVoidResult { get; }
```

Comment:
can we hide `IsVoidResult`? how is it supposed to be used given that `TResult` can't be void?